subscriptions/models.py

```diff
@@ -9,9 +11,39 @@
 from kukkuu.consts import OCCURRENCE_IS_NOT_FULL_ERROR
 from subscriptions.consts import NotificationType
 
+if TYPE_CHECKING:
+    from users.models import User
+
 
 class FreeSpotNotificationSubscriptionQueryset(models.QuerySet):
-    def user_can_view(self, user):
+    def user_subscriptions(
+        self, user: "User", child: Optional[Child] = None
+    ) -> Union[models.QuerySet, list["FreeSpotNotificationSubscription"]]:
+        """Get user's free spot notification subcriptions
+
+        Args:
+            user (User): an user instance
+            child (Optional[Child], optional): a child instance. Defaults to None.
+
+        Raises:
+            ValueError: if the user is not a guradian for the child.
+
+        Returns:
+            Queryset: list of user's FreeSpotNotificationSubscription instances,
```

Comment:
Same here as in https://github.com/City-of-Helsinki/kukkuu/pull/347/files#r1537394536